server/src/main/java/com/showmetheroapes/paracord/services/StrandService.java

```diff
@@ -22,7 +32,43 @@ public StrandService(StrandRepository strandRepository) {
 
   public StrandModel createStrand(Request request) {
     StrandModel strand =
-        new StrandModel(request.getName(), request.getIpAddress(), request.getPort());
+        StrandModel.builder()
+            .name(request.getName())
+            .ipAddress(request.getIpAddress())
+            .port(request.getPort())
+            .build();
     return strandRepository.save(strand);
   }
+
+  // Run strands check every 10 minutes
```

Comment:
I'd be wary of commenting implementation specific things like this - they turn into lies very easily, especially whenever this thing ideally becomes an environment variable